README.md

```diff
@@ -3,16 +3,33 @@
 ![TypeGPU (light mode)](./apps/typegpu-docs/public/typegpu-logo-light.svg#gh-light-mode-only)
 ![TypeGPU (dark mode)](./apps/typegpu-docs/public/typegpu-logo-dark.svg#gh-dark-mode-only)
 
-[Website](https://docs.swmansion.com/typegpu) — [Live Examples](https://docs.swmansion.com/typegpu/examples) — [Documentation](https://docs.swmansion.com/typegpu/guides/getting-started)
+[Website](https://docs.swmansion.com/typegpu) — [Documentation](https://docs.swmansion.com/typegpu/guides/getting-started)
 
 </div>
 
-**TypeGPU** is a TypeScript library simplifying the WebGPU API and WGSL with zero-cost abstractions, type-safe data transfer and dependency injection.
+**TypeGPU** is a TypeScript library that enhances the WebGPU API, allowing resource management in a type-safe, declarative way.
+
+<div align="center">
+<video muted loop autoplay controls src="./apps/typegpu-docs/public/assets/migration.mp4">
```

Comment:
doesn't work on github. I think maybe putting just a url would work (like here: https://github.com/alelievr/Mixture/blob/0.4.0/README.md), but we would need to try it out